massa-pos-worker/src/lib.rs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2022 MASSA LABS <info@massa.net>
+
+mod controller;
+mod worker;
+
+use massa_pos_exports::CycleInfo;
+
+use parking_lot::{Condvar, Mutex};
+use std::{collections::VecDeque, sync::Arc};
+
+/// Same structure pointer that will be used by the selector controller and his
+/// thread. It will store all new CycleInfo declared by massa (in the
+/// Execution module) and will be used to compute the draws in background.
+pub(crate) type InputDataPtr = Arc<(Condvar, Mutex<VecDeque<CycleInfo>>)>;
```

Comment:
I find declaring new types to not add value, and only make you have to look-up what that type is. Why not just pass the `Arc<_>` around?